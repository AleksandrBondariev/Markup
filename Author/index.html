<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Project Title</title>
	<link rel="stylesheet" href="css/owl.carousel.css">
	<link rel="stylesheet" href="css/owl.theme.default.css">
	<link rel="stylesheet" href="css/font-awesome.min.css">
	<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600" rel="stylesheet">
	<link media="all" rel="stylesheet" href="css/main.css">
</head>
<body>
	<header id="header">
		<div class="container">
			<nav class="nav">
				<a href="#" class="nav-opener"><span>Menu</span></a>
				<div class="drop">
					<ul>
						<li><a href="#" class="active">Home</a></li>
						<li><a href="#">Service</a></li>
						<li><a href="#">Portfolio</a></li>
						<li><a href="#">Team</a></li>
						<li><a href="#">Blog</a></li>
						<li><a href="#">Contact</a></li>
					</ul>
				</div>
			</nav>
			<a href="#" class="logo"><img src="images/logo.png" width="120" height="40" alt="Author"></a>
		</div>
	</header>
	<main id="main">
		<div class="owl-carousel owl-theme">
		    <div class="item-owl">
		    	<img src="images/slide.jpg" alt="">
				<div class="item">
		    		<h2>This is a main heading</h2>
		    		<p>Lorem Ipsum is simply dummy text of the printing</p>
		    		<a href="#" class="btn">Learn More</a>
		    	</div>
		    </div>
		    <div class="item-owl">
		    	<img src="images/slide.jpg" alt="">
				<div class="item">
		    		<h2>This is a main heading</h2>
		    		<p>Lorem Ipsum is simply dummy text of the printing</p>
		    		<a href="#" class="btn">Learn More</a>
		    	</div>
		    </div>
		    <div class="item-owl">
		    	<img src="images/slide.jpg" alt="">
				<div class="item">
		    		<h2>This is a main heading</h2>
		    		<p>Lorem Ipsum is simply dummy text of the printing</p>
		    		<a href="#" class="btn">Learn More</a>
		    	</div>
		    </div>
		</div>
		<div class="about-team container">
			<h3>The Team</h3>
			<div class="four-colums">
				<div class="col">
					<div class="wrapp-box">
						<img src="images/Ellipse.png" alt="team">
					</div>
					<h4>Adrian Thomas</h4>
					<p>CEO & Founder</p>
					<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been </p>
				</div>
				<div class="col">
					<div class="wrapp-box">
						<img src="images/Ellipse.png" alt="team">
						<a href="#" class="plus"><i class="fa fa-plus" aria-hidden="true"></i></a>
					</div>
					<h4>Narate Ketram</h4>
					<p>Creative Director</p>
					<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been </p>
				</div>
				<div class="col">
					<div class="wrapp-box">
						<img src="images/Ellipse.png" alt="team">
					</div>
					<h4>Fernando Comet</h4>
					<p>CEO & Founder</p>
					<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been </p>
				</div>
				<div class="col">
					<div class="wrapp-box">
						<img src="images/Ellipse.png" alt="team">
					</div>
					<h4>Adedayo saheed</h4>
					<p>Creative Director</p>
					<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been </p>
				</div>
			</div>
		</div>
		<div class="offers">
			<div class="container">
				<div class="three-columns">
					<div class="col">
						<div class="box-basic">
							<div id="basic-hover" class="box-header" onmouseover="changeDivBasic()" onmouseout="defaultDivBasic()">Basic</div>
							<div class="box-price">$5.01/ Month</div>
							<div class="box-main">Lorem ipsum</div>
							<div class="box-main">Dolor sit amet, consect</div>
							<div class="box-main">Adipiscing elit</div>
							<div class="box-main">Proin commodo turpis</div>
							<div class="box-main">lacus pulvinarvel </div>
							<div class="box-main">Prnare nisi pretium.</div>
							<div class="box-fotter"><a href="#" id="basic-btn" class="btn" onmouseover="changeDivBasic()" onmouseout="defaultDivBasic()">Buy now</a></div>
						</div>
					</div>
					<div class="col">
						<div class="box-pro">
							<div id="pro-hover" class="box-header" onmouseover="changeDivPro()" onmouseout="defaultDivPro()">Pro</div>
							<div class="box-price">$10.01/ Month</div>
							<div class="box-main">Lorem ipsum</div>
							<div class="box-main">Dolor sit amet, consect</div>
							<div class="box-main">Adipiscing elit</div>
							<div class="box-main">Proin commodo turpis</div>
							<div class="box-main">lacus pulvinarvel </div>
							<div class="box-main">Prnare nisi pretium.</div>
							<div class="box-fotter"><a href="#" id="pro-btn" class="btn" onmouseover="changeDivPro()" onmouseout="defaultDivPro()">Buy now</a></div>
						</div>
					</div>
					<div class="col">
						<div class="box-premium">
							<div id="premium-hover" class="box-header" onmouseover="changeDivPremium()" onmouseout="defaultDivPremium()">Premium</div>
							<div class="box-price">$20.01/ Month</div>
							<div class="box-main">Lorem ipsum</div>
							<div class="box-main">Dolor sit amet, consect</div>
							<div class="box-main">Adipiscing elit</div>
							<div class="box-main">Proin commodo turpis</div>
							<div class="box-main">lacus pulvinarvel </div>
							<div class="box-main">Prnare nisi pretium.</div>
							<div class="box-fotter"><a href="#" id="premium-btn" class="btn" onmouseover="changeDivPremium()" onmouseout="defaultDivPremium()">Buy now</a></div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</main>
	<div id="footer">
		<div class="contact-us container">
			<h3>Contact Us</h3>
			<div class="two-columns">
				<div class="col">
					<div class="box-contact-us">
						<h4>Get in touch with us</h4>
						<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's </p>
						<ul class="social-networks">
							<li><a class="facebook" href="#"><i class="fa fa-facebook"></i></a></li>
					      <li><a class="twitter" href="#"><i class="fa fa-twitter"></i></a></li>
				          <li><a class="google-plus" href="#"><i class="fa fa-google-plus"></i></a></li>
						</ul>
						<div class="box-information">
							<p>Posted by Dospel & GanjaParker</p>
							<a href="tel:+XXXXXXXXXXXX">+XX-XXXX - XXXX - XX</a>
							<a href='&#109;a&#105;&#108;&#116;o&#58;%69nfo&#64;m&#97;%69&#108;&#46;c&#111;&#37;&#54;D'>info&#64;mai&#108;&#46;c&#111;&#109;</a>
						</div>
					</div>
				</div>
				<div class="col">
					<form action="#" class="contact-form">
			        	<input id="name" name="name" minlength="2" type="text" required placeholder="Name">
			        	<input id="email" type="email" name="email" required placeholder="Email">
			        	<textarea  id="textarea" name="comment" required minlength="2" cols="60" rows="10" placeholder="Message"></textarea>
			        	<input type="submit" value="Send">
					</form>
				</div>
			</div>
		</div>
	</div>
<script type="text/javascript" src="js/jquery-1.11.2.min.js"></script>
<script src="js/jquery.main.js"></script>
<script src="js/jquery.validate.min.js"></script>
<script src="js/additional-methods.min.js"></script>
<script src="js/owl.carousel.js"></script>
<script>
	jQuery(document).ready(function($) {
        $(".owl-carousel").owlCarousel({
		    items:1,
		    nav:true,
		    dots: false,
		    autoplay:true,
		    responsiveClass:true,
		    navText: ["<i class='fa fa-chevron-left'></i>","<i class='fa fa-chevron-right'></i>"],
		        responsive:{
        0:{
            nav:false
        },
        600:{
            nav:false
        },
        1000:{
            nav:true
        }
    }
        });
    });
</script>
<script>
function changeDivBasic(){
  document.getElementById("basic-hover").style.backgroundColor="red";
  document.getElementById("basic-btn").style.borderColor="red";  
}
function defaultDivBasic(){
  document.getElementById("basic-hover").style.backgroundColor="";
  document.getElementById("basic-btn").style.borderColor="";  
}
function changeDivPro(){
  document.getElementById("pro-hover").style.backgroundColor="red";
  document.getElementById("pro-btn").style.borderColor="red";  
}
function defaultDivPro(){
  document.getElementById("pro-hover").style.backgroundColor="";
  document.getElementById("pro-btn").style.borderColor="";  
}
function changeDivPremium(){
  document.getElementById("premium-hover").style.backgroundColor="red";
  document.getElementById("premium-btn").style.borderColor="red";  
}
function defaultDivPremium(){
  document.getElementById("premium-hover").style.backgroundColor="";
  document.getElementById("premium-btn").style.borderColor="";  
}
</script>
</body>
</html>
