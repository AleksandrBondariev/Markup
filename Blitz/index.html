<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Blitz</title>
	<link media="all" rel="stylesheet" href="css/main.css">
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
</head>
<body>
	<div id="wrapper">
		<div class="top-bar">
			<div class="conteiner">
				<strong class="welcome"><p>Welcome to our website!</p></strong>
				<strong class="log-in"><p>Have an account? <a href="#">Log in</a> or <a href="#">Sign up</a></p></strong>
			</div>
		</div>
		<header id="header">
			<div class="conteiner">
				<a href="#" class="logo"><img src="sourceimages/logo.png" alt="Insight"></a>
				<strong class="slogan"> Multipurpose PSD Template</strong>
				<nav id="nav">
					<ul>
						<li class="active" ><a href="#" >HOME</a>							
							<ul>
								<li><a href="#">ABOUT US</a></li>
								<li class="active"><a href="#" >FEATURED SERVICES</a></li>
								<li><a href="#">OUR TEAM</a></li>
								<li><a href="#">OUR HISTORY</a></li>
								<li><a href="#">SECOND MENU ITEM</a></li>
							</ul></li>
						<li><a href="#">FEATURES</a></li>
						<li><a href="#">BLOG</a></li>
						<li><a href="#">PORTFOLIO</a></li>
						<li><a href="#">CONTACT</a></li>
					</ul>
				</nav>
			</div>
	</header>
		<div class="page-title">
			<div class="conteiner">
				<h1>CONTACT US</h1>
					<nav class="breadcrumbs">
						<ul>
							<li><a href="#">Home</a></li>
							<li><a href="#">Features</a></li>
							<li>Contact Us</li>
					</ul>
				</nav>
			</div>
	 	</div>
	<main id="main">
		<div class="conteiner">
			<div class="too-colums">
				<div class="columns">
					<h2>OUR LOCATION</h2>
						<div class="location">
							<img src="sourceimages/map.png" alt="Map">
						</div>
					<h2>CONTACT INFO</h2>
					<div class="contact-info">
						<address>Envato Pty Ltd, 89 Third Main Street, Miami, USA</address>
						<dl>
							<dt>Phone:</dt>
							<dd>+1 800 123 4567</dd>
							<dt>Fax:</dt>
							<dd>+1 800 123 45 67</dd>
							<dt>E-mail:</dt>
							<dd><a href="mailto:&#100;&#101;&#109;&#111;&#64;&#121;&#111;&#117;&#114;&#115;&#105;&#116;&#101;&#46;&#99;&#111;&#109;">&#100;&#101;&#109;&#111;&#64;&#121;&#111;&#117;&#114;&#115;&#105;&#116;&#101;&#46;&#99;&#111;&#109;</a></dd>
							<dt>Web:</dt>
							<dd><a href="http://smartchoice.com">www.smartchoice.com</a></dd>
						</dl>
					</div>
				</div>
				<div class="columns">
					<h2>CONTACT FORM</h2>
					<form action="#" class="contact-form">
					<label for="name">Your Name <span>(required)</span></label>
					<div class="row">
		                    <input id="name" type="text">
		             </div>
		             <label for="email">E-mail <span>(required)</span></label>
		              <div class="row">
		                    <input id="email" type="email">
		             </div>
		             <label for="website">Website</label>
		             <div class="row">
		                    <input id="url" type="text">
		              </div>
		              <label for="textarea">Your Message <span>(required)</span></label>
		                <div class="row">
		                    <textarea  id="textarea" cols="60" rows="10"></textarea>
		              </div>
		              	<input type="submit" value="SEND MESSAGE">
					</form>
				</div>
			</div>
		</div>
	</main>
		<aside class="bottom-section">
			<div class="conteiner">
				<div class="four-colums">
				<div class="col">
					<h3>SOCIAL NETWORKS</h3>
					<ul class="social-networks">
				      <li><a class="twitter" href="#"><i class="fa fa-twitter"></i></a></li>
			          <li><a class="facebook" href="#"><i class="fa fa-facebook"></i></a></li>
			          <li><a class="google-plus" href="#"><i class="fa fa-google-plus"></i></a></li>
			          <li><a class="dribbble" href="#"><i class="fa fa-dribbble"></i></a></li>
			          <li><a class="rss" href="#"><i class="fa fa-rss"></i></a></li>
					</ul>
						<h3>NEED AN ANSWER?</h3>
						<a class="phone" href="tel:1-900-1234-5678">1-900-1234-5678</a>
				</div>
				<div class="col">
					<h3>TWITTER UPDATES</h3>
					<ul class="twitter-update">
						<li><a href="#">@danfisher_dev</a> Pellentesque eget ante massa, eu interdum turpis. Nulla facilisi eros, at congue :)<br><time class="posted">2 days ago</time></li>
						<li><a href="#">@danfisher_dev</a> Nulla sed dolor ligula, in egestas ipsum. In orci nisi, consectetur nisl massa ut metus.<br> <time class="posted">2 days ago</time></li>
					</ul>
				</div>
				<div class="col">
					<h3>FORN THE BLOG</h3>
					<ul class="blog">
						<li><a href="#">Lorem ipsum dolor sit amet, consectetur</a></li>
						<li><p><a href="#">Aenean justo mauris, rhoncus nec tempor convallis</a></p></li>
						<li><p><a href="#">Nulla euismod ullamcorper tortor,  sed aliquam</a></p></li>
					</ul>


				</div>
				<div class="col">
					<h3>FLICKR</h3>
					<ul class="flickr">
						<li><a href=""><img  src="sourceimages/cube.png" alt="Cube"></a></li>
						<li><a href=""><img  src="sourceimages/cube.png" alt="Cube"></a></li>
						<li><a href=""><img  src="sourceimages/cube.png" alt="Cube"></a></li>
						<li><a href=""><img  src="sourceimages/cube.png" alt="Cube"></a></li>
						<li><a href=""><img  src="sourceimages/cube.png" alt="Cube"></a></li>
						<li><a href=""><img  src="sourceimages/cube.png" alt="Cube"></a></li>
					</ul>
				</div>
			</div>		
		</div>
	</aside>
<footer id="footer">
	<div class="conteiner">
		<p>&copy; 2013 Blitz | Multipurpose PSD Template</p>
		</div>
</footer>
	</div>
	<script src="js/jquery-1.11.2.min.js"></script>
	<script src="js/jquery.main.js"></script>
</body>
</html>
