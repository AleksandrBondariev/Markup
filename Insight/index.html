<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>Insight</title>
	<link media="all" rel="stylesheet" href="css/main.css">
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
</head>
<body>
	<div id="wrapper">
			<header id="header">
					<div class="conteiner">
						<nav id="nav">
							<ul>
								<li><a href="#"><i class="fa fa-question-circle"></i>  How it works</a></li>
								<li class="active"><a href="#"><i class="fa fa-user"></i>  Sign up</a></li>
								<li><a href="#"><i class="fa fa-lock"></i>  Login</a></li>
							</ul>
						</nav>
							<a href="#" class="logo"><img src="sourceimages/logo.png" alt="Insight"></a>
							<strong class="slogan">your nice slogan</strong>
					</div>
			</header>
		<div class="visual">
			<div class="conteiner">
			        <div class="intro">
						<h1>Introducing Something Cool</h1>
						<p>I would like to avoid making these mistakes. But how do you avoid mistakes you make by default? Ideally you transform your life so it has other defaults. lorem*20</p>
	                </div>
		            <div class="photo">
						<img src="sourceimages/main-phone.png" height="460px" width="333px" alt="Phone">
		              </div>
	            <div class="form-sign-up">
					<label for="name">New to Insight? <b>Sign up here:</b></label>
						<form action="#" class="contact-form">
		                    <input id="name" type="text"  placeholder="Name">
		                    <input id="email" type="email" value=""placeholder="Email">
		                    <input id="password" type="password" placeholder="Password">       
		              </form>
		              <p>By creating an account, I accept Insightâ€™s <a href="#">Terms of Service</a> and <a href="#">Privacy Policy.</a></p>
		              <input class="btn" type="submit" value="Sing Up">
               </div>
		</div>
	</div>
		<main class="conteiner">
			<div class="three-columns">
			<div class="columns">
				<img src="sourceimages/Shape.png" width="274" height="127" alt="">
				<h3>Top of my to do list</h3>
				<p>I would like to avoid making these mistakes. But how do you avoid mistakes you make by default? Ideally you transform your life so it has other defaults.</p>
				<a href="#"class="btn">Learn more</a>
			</div>
			<div class="columns">
				<img src="sourceimages/Shape2.png" height="127" width="274" alt="">
				<h3>The alarming thing</h3>
				<p>I would like to avoid making these mistakes. But how do you avoid mistakes you make by default? Ideally you transform your life so it has other defaults.</p>
				<a href="#"class="btn">Learn more</a>
			</div>
			<div class="columns">
				<img src="sourceimages/Shape3.png" height="127" width="274" alt="">
				<h3>Errors of omission</h3>
				<p>I would like to avoid making these mistakes. But how do you avoid mistakes you make by default? Ideally you transform your life so it has other defaults.</p>
				<a href="#"class="btn">Learn more</a>
			</div>
		</div>
			<div class="features">
				<p>There are a lot of great reasons to use Insight.<a href="#"> Check out our features <i class="fa fa-chevron-right"></i></a></p>
			</div>
	</main>
	<footer id="footer">
		<div class="conteiner">
			<div class="col-footer">
				<h3>MAIN</h3>
				<ul>
					<li><a href="#">Start Here </a></li>
					<li><a href="#">Portfolio</a></li>
					<li><a href="#">Meet Us</a></li>
					<li><a href="#">Blog</a></li>
					<li><a href="#">Contact</a></li>
				</ul>
			</div>
			<div class="col-footer">
				<h3>COMPANY</h3>
				<ul>
					<li><a href="#">About</a></li>
					<li><a href="#">Help</a></li>
					<li><a href="#">Support</a></li>
					<li><a href="#">Jobs</a></li>
					<li><a href="#">Directory</a></li>
				</ul>
			</div>
			<div class="col-footer">
				<h3>ONE MORE</h3>
				<ul>
					<li><a href="#">Meetups</a></li>
					<li><a href="#">Handbook</a></li>
					<li><a href="#">Privacy</a></li>
					<li><a href="#">API</a></li>
					<li><a href="#">Equipment</a></li>
				</ul>
			</div>
			<div class="col-footer">
				<h3>THE LAST ONE</h3>
				<ul>
					<li><a href="#">Meetups</a></li>
					<li><a href="#">Handbook</a></li>
					<li><a href="#">Privacy</a></li>
					<li><a href="#">API</a></li>
					<li><a href="#">Equipment</a></li>
				</ul>
			</div>
				<ul class="socila-networks">
					<li><a href="#"><i class="fa fa-twitter"></i></a></li>
					<li><a href="#"><i class="fa fa-facebook"></i></a></li>
					<li><a href="#"><i class="fa fa-vimeo"></i></a></li>
				</ul>

				<p>&copy; 2013 be happy</p>
					
				<ul class="footer-links">
					<li><a href="#">Privacy Policy</a></li>
					<li><a href="#">Terms of Service</a></li>
				</ul>
		</div>
</footer>
</div>
	<script src="js/jquery-1.11.2.min.js"></script>
	<script src="js/jquery.main.js"></script>
</body>
</html>
