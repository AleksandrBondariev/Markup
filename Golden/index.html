<!DOCTYPE html>
<html lang="en">
<head>
	<meta http-equiv="Content-type" content="text/html; charset=utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
	<title>Portfolio</title>
	<link href="css/style.css" type="text/css" rel="stylesheet" />
    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Oswald:400,300" type="text/css">
</head>
<body>
<div class="wrapper">
	<div id="header">
		<div class="conteiner clearfix">
				<div class="logo"><a href="#"><img src="img/logo.png" height="28" width="79" alt="Логотип"></a>
			</div>
			<ul class="nav">
				<li><a href="#">Home</a></li>
				<li><a href="#">Services</a></li>
				<li><a href="#">Portfolio</a></li>
				<li><a href="#">About</a></li>
				<li><a href="#">Contacts</a></li>
			</ul>
		</div>
	</div>
	<div id="home">
		<div class="conteiner">
		<h3>Welcome To Our Studio!</h3>
		<h1>It’s nice to Meet you</h1>
		<buttun id="btn">Tell Me More</buttun>
		</div>
	</div>
	<div id="services" class="box">
		<div class="conteiner">
			<h2 class="title">Services</h2>
			<p class="descr">Proin iaculis purus consequat sem cure. </p>
			<div class="cervices-wrap clearfix">
				<div class="services-item commerce">
					<h5 class="title">E-Commerce</h5>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit</p>
				</div>
				<div class="services-item responsive">
					<h5 class="title">Responsive Web</h5>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit</p>
				</div>
				<div class="services-item secutity">
					<h5 class="title">Web Security</h5>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit</p>
				</div>
			</div>
		</div>
	</div>
		<div id="portfolio" class="box">
		<div class="conteiner">
			<h2 class="title">Our portfolio</h2>
			<p class="descr">Proin iaculis purus consequat sem cure. </p>
			<div class="portfolio-wrap clearfix">
				<div class="portfolio-item">
					<a href=""><img src="img/ebony.png" alt=""></a>
					<div class="p-item-footer">
						<h5 class="title">Ebony &amp; Ivory</h5>
						<span>Branding</span>
					</div>
				</div>
						<div class="portfolio-item">
					<a href=""><img src="img/smart.jpg" alt=""></a>
					<div class="p-item-footer">
						<h5 class="title">Smart Stationary</h5>
						<span>Print Design</span>
					</div>
				</div>
						<div class="portfolio-item">
					<a href=""><img src="img/clever.jpg" alt=""></a>
					<div class="p-item-footer">
						<h5 class="title">Clever Poster </h5>
						<span>Print Design</span>
					</div>
				</div>
						<div class="portfolio-item">
					<a href=""><img src="img/vinyl.jpg" alt=""></a>
					<div class="p-item-footer">
						<h5 class="title">Vinyl Record </h5>
						<span>Product Mock-Up</span>
					</div>
				</div>
						<div class="portfolio-item">
					<a href=""><img src="img/treehouse.jpg" alt=""></a>
					<div class="p-item-footer">
						<h5 class="title">Treehouse Template</h5>
						<span>Web Design</span>
					</div>
				</div>
						<div class="portfolio-item">
					<a href=""><img src="img/burned.jpg" alt=""></a>
					<div class="p-item-footer">
						<h5 class="title">Burned Logo</h5>
							<span>Branding</span>
							</div>
						</div>
					</div>
				</div>
				<div id="about" name="about" class="box">
				<div class="conteiner ">
					<h2 class="title">About Us</h2>
					<p class="descr">Proin iaculis purus consequat sem cure. </p>
					<div class="about-wrap">
						<div class="about-item clearfix">
							<div class="about-text  left text-right">
								<h5 class="title"><span>July 2010</span> <br>
								Our Humble Beginnings</h5>
								<p>Proin iaculis purus consequat sem cure 
									digni ssim. Donec porttitora entum suscipit 
									aenean rhoncus posuere odio in tincidunt. Proin 
									iaculis purus consequat sem cure digni 
									ssim. Donec porttitora entum suscipit.</p>
							</div>
							<div class="about-img left">
								<img src="img/about1.png" alt="Картинка">
							</div>
						</div>
								<div class="about-item clearfix">
							<div class="about-text  right text-left">
								<h5 class="title"><span>January 2011</span> <br>
												Facing Startup Battles
												</h5>
								<p>Proin iaculis purus consequat sem cure 
									digni ssim. Donec porttitora entum suscipit 
									aenean rhoncus posuere odio in tincidunt. Proin 
									iaculis purus consequat sem cure digni 
									ssim. Donec porttitora entum suscipit aenean 
									rhoncus posuere odio in tincidunt.</p>
							</div>
							<div class="about-img right">
								<img src="img/about2.png" alt="Картинка2">
							</div>
						</div>
								<div class="about-item clearfix">
							<div class="about-text  left text-right">
								<h5 class="title"><span>December 2012</span> <br>
									Enter The Dark Days
									</h5>
								<p>Proin iaculis purus consequat sem cure 
									digni ssim. Donec porttitora entum suscipit 
									aenean rhoncus posuere odio in tincidunt. Proin 
									iaculis purus consequat sem cure digni 
									ssim. Donec porttitora entum suscipit.</p>
							</div>
							<div class="about-img left">
								<img src="img/about3.png" alt="Картинка3">
							</div>
						</div>
						<div class="about-item clearfix">
							<div class="about-text  right text-left">
								<h5 class="title"><span>february 2014</span> <br>
												Our Triumph
												</h5>
								<p>Proin iaculis purus consequat sem cure 
									digni ssim. Donec porttitora entum suscipit 
									aenean rhoncus posuere odio in tincidunt. Proin 
									iaculis purus consequat sem cure digni 
									ssim. Donec porttitora entum suscipit.</p>
							</div>
							<div class="about-img right">
								<img src="img/about4.png" alt="Картинка4">
							</div>
						</div>
						<a href="#" class="about-link">Our <br />Story <br />Continues <br />...</a>
					</div>
					</div>
				</div>
				<div id="team"  class="box">
					<div class="conteiner">
						<h2 class="title">our amazing team</h2>
						<p class="descr">Proin iaculis purus consequat sem cure. </p>
						<div class="team-wrap clearfix">
							<div class="team-item">
								<img src="img/team-img.png" height="220" width="220" alt="Фото">
								<h5 class="title">Kimberly Thompson</h5>
								<p clsss="cat">Marketer</p>
								<div class="social">
								<a href="#" class="ic-tw">&nbsp;</a>
								<a href="#" class="ic-fb">&nbsp;</a>
								<a href="#" class="ic-pn">&nbsp;</a>
								</div>	
							</div>
							<div class="team-item">
								<img src="img/team-img.png" height="220" width="220" alt="Фото">
								<h5 class="title">Kimberly Thompson</h5>
								<p clsss="cat">Marketer</p>
								<div class="social">
								<a href="#" class="ic-tw">&nbsp;</a>
								<a href="#" class="ic-fb">&nbsp;</a>
								<a href="#" class="ic-gl">&nbsp;</a>
								</div>	
							</div>
							<div class="team-item">
								<img src="img/team-img.png" height="220" width="220" alt="Фото">
								<h5 class="title">Kimberly Thompson</h5>
								<p clsss="cat">Marketer</p>
								<div class="social">
								<a href="#" class="ic-fb">&nbsp;</a>
								<a href="#" class="ic-pn">&nbsp;</a>
								</div>	
							</div>
						</div>
						<p class="descr team-info">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit <br>  
						aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
					</div>	
				</div>
				<div id="brand">
					<div class="conteiner">
						<a href="#"><img src="img/envato.png" width="161" height="30" alt="Картинка"></a>
						<a href="#"><img src="img/press.png" width="186" height="42" alt="Картинка"></a>
						<a href="#"><img src="img/tuts.png" width="131" height="41" alt="Картинка"></a>
						<a href="#"><img src="img/microlancer.png" width="167" height="42" alt="Картинка"></a>
					</div>
				</div>
				<div id="contact" name="contact"  class="box">
					<div class="conteiner">
						<h2 class="title">Contact us</h2>
						<p class="descr">Proin iaculis purus consequat sem cure. </p>
						<form action="#" id="masseges">
							<div class="clearfix">
								<div class="left">
									<input type="text" placeholder="your Name *" />
									<input type="text" placeholder="your e-mail *" />
									<input type="text" placeholder="subject" />
								</div>
								<div class="right">
									<textarea   cols="0" rows="0" placeholder="Your message *" >
									</textarea>
								</div>
							</div>
							<input type="submit" value="Send Message" id="btn"  />
					</form>

				</div>
			</div>
		</div>
		<div id="copyright">
			<div class="conteiner  clearfix">
				<p class="left">&copy;Copyright 2014 FreebiesXpress.com</p>
				<div class="social footer" id="social">
					<a href="#" class="ic-tw black">&nbsp;</a>
					<a href="#" class="ic-fb black">&nbsp;</a>
					<a href="#" class="ic-pn black">&nbsp;</a>
					<a href="#" class="ic-gl black">&nbsp;</a>
				</div>	
			</div>
		</div>
	</div>
</body>
</html>